MobileWallet/Screens/Home/Transaction/TransactionViewController.swift

```diff
@@ -588,6 +588,22 @@ extension TransactionViewController: UITextFieldDelegate {
             return true
         }
 
+        if newAlias.trimmingCharacters(in: .whitespaces).isEmpty {
+            UIImpactFeedbackGenerator(style: .light).impactOccurred()
+            if newAlias.isEmpty {
+                textField.endEditing(true)
+                do {
+                    if let contact = try TariLib.shared.tariWallet!.contacts.0?.find(publicKey: contactPublicKey!) {
+                        try TariLib.shared.tariWallet!.removeContact(contact)
```

Comment:
Don't force unwrap wallet, we are stopping and starting the wallet service when the app moves to the background. It can be nil for a few seconds before starting and we don't want to crash the app each time.